Makefile

```diff
@@ -79,122 +105,137 @@ menuconfig: Kconfig
+guestagents: $(GUESTAGENTS)
+native-guestagent: $(NATIVE_GUESTAGENT)
+additional-guestagents: $(ADDITIONAL_GUESTAGENTS)
+%-guestagent:
+	@[ "$(findstring $(*),$(GUESTAGENT_ARCHS))" == "$(*)" ] && make _output/share/lima/lima-guestagent.Linux-$*
+_output/share/lima/lima-guestagent.Linux-%: | _output/share/lima
+	GOOS=linux $(GUESTAGENT_ARCH_ENVS_$*) CGO_ENABLED=0 $(GO_BUILD) -o $@ ./cmd/lima-guestagent
+	chmod 644 $@
+ifeq ($(CONFIG_GUESTAGENT_COMPRESS),y)
+	gzip $@
 endif
-	echo $(VERSION) > _output/share/doc/lima/VERSION
 
-.PHONY: _output/bin/lima
-_output/bin/lima:
-	mkdir -p _output/bin
-	cp -a ./cmd/lima $@
+MKDIR_TARGETS += _output/share/lima
 
-.PHONY: _output/bin/lima.bat
-_output/bin/lima.bat:
-	mkdir -p _output/bin
-	cp -a ./cmd/lima.bat $@
+# _output/share/lima/templates
+TEMPLATES=$(addprefix _output/share/lima/templates/,$(filter-out experimental,$(notdir $(wildcard examples/*))))
+TEMPLATE_EXPERIMENTALS=$(addprefix _output/share/lima/templates/experimental/,$(notdir $(wildcard examples/experimental/*)))
 
-.PHONY: _output/bin/nerdctl.lima
-_output/bin/nerdctl.lima:
-	mkdir -p _output/bin
-	cp -a ./cmd/nerdctl.lima $@
+.PHONY: default_template templates template_experimentals
+default_template: _output/share/lima/templates/default.yaml
+templates: $(TEMPLATES)
+template_experimentals: $(TEMPLATE_EXPERIMENTALS)
 
-_output/bin/apptainer.lima: ./cmd/apptainer.lima
-	@mkdir -p _output/bin
-	cp -a $^ $@
+$(TEMPLATES): | _output/share/lima/templates
+$(TEMPLATE_EXPERIMENTALS): | _output/share/lima/templates/experimental
+MKDIR_TARGETS += _output/share/lima/templates _output/share/lima/templates/experimental
 
-_output/bin/docker.lima: ./cmd/docker.lima
-	@mkdir -p _output/bin
-	cp -a $^ $@
+_output/share/lima/templates/%: examples/%
+	cp -aL $< $@
 
-_output/bin/podman.lima: ./cmd/podman.lima
-	@mkdir -p _output/bin
-	cp -a $^ $@
 
-_output/bin/kubectl.lima: ./cmd/kubectl.lima
-	@mkdir -p _output/bin
-	cp -a $^ $@
+# _output/share/lima/examples
+.PHONY: create-examples-link
+create-examples-link: _output/share/lima/examples
+_output/share/lima/examples: default_template # depends on minimal template target
+ifneq ($(GOOS),windows)
+	ln -sf templates $@
+else
+# copy from templates builded in build process
+	cp -aL _output/share/lima/templates $@
+endif
 
-.PHONY: _output/bin/limactl$(exe)
-_output/bin/limactl$(exe):
-	# The hostagent must be compiled with CGO_ENABLED=1 so that net.LookupIP() in the DNS server
-	# calls the native resolver library and not the simplistic version in the Go library.
-	CGO_ENABLED=1 $(GO_BUILD) -o $@ ./cmd/limactl
+# _output/share/doc/lima
+DOCUMENTS=$(addprefix _output/share/doc/lima/,$(wildcard *.md) LICENSE SECURITY.md VERSION)
 
-.PHONY: _output/share/lima/lima-guestagent.Linux-x86_64
-_output/share/lima/lima-guestagent.Linux-x86_64:
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO_BUILD) -o $@ ./cmd/lima-guestagent
-	chmod 644 $@
+.PHONY: documents
+documents: $(DOCUMENTS)
```

Comment:
The "documents" target should probably be called "documentation" ?

Reply:
Thanks!
Changed to `documentation`.
Also changed `create-links-in-documents` to `create-links-in-doc-dir`.

Reply:
Also changed `DOCUMENTS` variable name to `DOCUMENTATION`.